mockOSApi.v2/DTO/ProcessDto.cs

```diff
@@ -9,28 +8,23 @@ public class PriorityClass()
     public int Pid { get; set; }
 }
 
-
-
 public class MockProcessDto()
 {
     public int Pid { get; set; }
     public string? Image { get; set; }
-
-    public ProcState Status { get; set; }
-
+    public ProcessStatus Status { get; set; }
     public int Priority { get; set; }
-
+    public List<int> FileDescriptors { get; set; }
+    public bool? IsService { get; set; }
+    public string? ErrorMessage { get; set; }
+    public User User { get; set; }
```

Comment:
Hmmm. I don't think you should have an actual user on the Dto.
A UserDto instead

Reply:
done

Reply:
done